### PR TITLE
Issue #44 print fields in alphabetical order, like flags

### PR DESCRIPTION
**Pull request by client9**
this is some minor cut-n-paste on the map[string]... to []string, but I didn't feel it was worth writing a function.  Feel free to adjust.

regards,

n

### CODE REVIEW

**Comment by client9** [comment]
Added fix for #46 as well.

File: env.go [hunk 1]
```diff
@@ -83,10 +84,15 @@ func (e *EnvironmentLoader) PrintEnvs(s interface{}) {
 	strctMap := strct.Map()
 	prefix := e.getPrefix(strct)
 
-	for key, val := range strctMap {
-		field := strct.Field(key)
+	keys := make([]string, 0, len(strctMap))
```
Comment (cihangir) [inline env.go]: just fyi. Structs package has `.Names()` function where it returns the field names.

**Comment by cihangir** [comment]
Thanks @client9 🎉

**Comment by client9** [comment]
oh awesome.

I can redo for clarity if you want.

```
Structs package has .Names() function where it returns the field names.!!
```

**Comment by cihangir** [comment]
It wont clean up much code so not needed, thanks for asking.

**Pull request merged**